smoketest/systemlimits_test.go

```diff
@@ -22,11 +25,11 @@ func TestSystemLimits(t *testing.T) {
 		values
 			({{uuid "cm_user"}}, 'CM User'),
 			({{uuid "nr_user"}}, 'NR User'),
-			({{uuid "generic_user1"}}, 'User 1'),
-			({{uuid "generic_user2"}}, 'User 2'),
-			({{uuid "generic_user3"}}, 'User 3'),
-			({{uuid "generic_user4"}}, 'User 4'),
-			({{uuid "generic_user5"}}, 'User 5');
+			('50322144-1e88-43dc-b638-b16a5be7bad6', 'User 1'),
+			('dfcc0684-f045-4a9f-8931-56da8a014a44', 'User 2'),
+			('016d5895-b20f-42fd-ad6c-7f1e4c11354d', 'User 3'),
+			('dc8416e1-bf15-4248-b09f-f9294adcb962', 'User 4'),
+			('c1dadc8b-b0fc-41e3-a015-5a14c5c19433', 'User 5');
```

Comment:
We should stick to using the UUID generators for consistency. They should be thread-safe. Additionally, the SQL query will be evaluated for each (e.g. used as template) so `generic_user1` will match `generic_user1` within each `t.Run`

Reply:
Yes, the reason why we can not use the UUID generators is because of the refactoring, the harness is only accessible within `t.Run()` and helper functions. More details in the next comment below.